BuildAutomation/BuildCloudbaseInitSetup.ps1

```diff
@@ -1,6 +1,7 @@
 Param(
   [string]$platform = "x64",
-  [string]$pythonversion = "3.11_9",
+  [string]$pythonversion = "3.12.3",
+  [string]$pythonversionPrelease = "",

```

Comment:
I assume ``Prelease`` stands for pre-release (e.g. ``b1``, ``a6``, etc), we could add a comment.

Reply:
Will fix the name, it s a typo